crates/wysiwyg/src/dom/nodes/mention_node.rs

```diff
@@ -41,8 +47,7 @@ where
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum MentionNodeKind {
-    Room { mention: Mention },
-    User { mention: Mention },
+    MatrixURI { mention: Mention },
```

Comment:
nitpick: In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn ([source](https://rust-lang.github.io/api-guidelines/naming.html))
```suggestion
    MatrixUri { mention: Mention },
```

Reply:
Addressed in [46f48cb](https://github.com/matrix-org/matrix-rich-text-editor/pull/719/commits/46f48cb0f9e7c353e673b4e1179333f54bd8342c)